types/migrations/20220714123910_edited_collections_2.sql

```diff
@@ -0,0 +1,6 @@
+-- Add migration script here
```

Comment:
- Could you rename this migration to something more unique like `edited_collection_intro_chapters` for example?
- You can get rid of this first line, it's a placeholder.